src/__tests__/index.ts

```diff
@@ -159,21 +159,6 @@ const tests: Record<string, TestCase> = {
     ],
     output: [{aliases: [{name: {first: 'baz'}},{name: {first: 'foo'}},{name: null}]}, {aliases: [{name: {first: 'foo'}},{name: {first: 'bat'}},null]}],
   },
-  'can handle object with an array of values with nested keys with an implicit wildcard': {
-    input: [
-      [
-        {aliases: [{name: {first: 'baz'}},{name: {first: 'foo'}},{name: null}]},
-        {aliases: [{name: {first: 'foo'}},{name: {first: 'bat'}},null]},
-        {aliases: [{name: {first: 'foo'}},{name: {first: 'foo'}}]},
-        {aliases: null},
-        {},
-        null,
-      ],
-      'ba',
-      {keys: ['aliases.name.first']},
```

Comment:
I don't want to support this. It's too much magic. So I'm glad that it's gone.

Reply:
Me too.